crates/core/Cargo.toml

```diff
@@ -30,10 +30,13 @@ petgraph = { version = "0.6", default-features = false, features = ["graphmap"]
 smallstr = { version = "0.3", features = ["union"] }
 smallvec = { version = "1.10", features = ["union", "const_generics", "specialization"] }
 thiserror = "1.0"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
```

Comment:
Minor formatting: please keep the dependencies sorted by name.

Reply:
I miss read this comment to say "format these using `version`" and then did all of them. I also sorted by name of course.